docs/tutorials/multimodal/advanced_topics/index.rst

```diff
@@ -15,10 +15,17 @@ Advanced Topics
 
       How to customize AutoMM configurations.
 
+   .. card::
+      :title: HPO in AutoMM
+      :link: hyperparameter_optimization.html
+
+      How to customize AutoMM configurations.
+   
 
 .. toctree::
    :maxdepth: 1
    :hidden:
 
    efficient_finetuning_basic
    customization
+   hyperparameter_optimization
```

Comment:
Do you think we should put it before customization ?

Reply:
I can move it, but I did mention `customization` in this tutorial to help readers on the search space.

Reply:
Or shall we move customization to be the first tutorial?

Reply:
Customization should be the last tutorial. Every tutorial can link to customization, and use it as a lookup table.

Reply:
Sounds good. I moved it before customization.